File: C++/1036 - Bhaskara's Formula.cpp
#include <stdio.h>
#include <iostream>
#include <cmath>
using namespace std;
 
int main() {
    double a, b, c, delta, resposta1, resposta2;
    
    cin >> a >> b >> c;
    
    delta = (b * b) - (4 * a * c);

    if (a == 0) {
        cout << "Impossivel calcular\n";
    } else if (delta < 0) {
        cout << "Impossivel calcular\n";
    } else {
        resposta1 = ((-b) + sqrt(delta)) / (2 * a);
        resposta2 = ((-b) - sqrt(delta)) / (2 * a);
        
        printf("R1 = %.5f\n", resposta1);
        printf("R2 = %.5f\n", resposta2);
    }
    
    return 0;
}

File: Python/1036 - Bhaskara's Formula.py
import math
a, b, c = map(float, input().split())

delta = b**2 - (4 * a * c)
if delta <= 0 or a == 0:
    print("Impossivel calcular")
else:
    print(f"R1 = {((-b + math.sqrt(delta)) / (2 * a)):.5f}")
    print(f"R2 = {((-b - math.sqrt(delta)) / (2 * a)):.5f}")
